marketplace_builder/views/pages/tutorials/integrations/twilio.liquid

```diff
@@ -0,0 +1,42 @@
+---
+converter: markdown
+metadata:
+  title: Integrating Twilio Using API Call Notifications
+  description: This guide will help you create a Twilio integration for your platformOS instance.
+slug: tutorials/integrations/twilio
+searchable: true
+---
+
+This guide will help you create a Twilio integration for platformOS using Partner Portal and SMS notifications.
+
+## Requirements
+To follow this tutorial, you should be familiar with creating SMS call notifications. You also have to have access to Partner Portal.
+
+* [Creating an SMS Notification](/tutorials/notifications/creating-sms-notification
+* [Twilio API Reference, REST API](https://www.twilio.com/docs/usage/api)
+* [Managing Integrations in Partner Portal](/tutorials/partner-portal/managing-integrations)
+## Steps
+
+Integrating SMS service with Twilio using SMS notifications is a three-step process:
+
+1. Create Twilio Account Using Partner Portal
+2. Add SMS notification to the form
+3. Billing
```

Comment:
https://github.com/mdyd-dev/nearme-documentation/pull/385#issue-251470321

Can you use automagic? ;)

Reply:
Added during review! Thank you!